### PR TITLE
Pull this!

### PR DESCRIPTION
**Pull request by Teylan**
haha yes

### CODE REVIEW

File: sign-in.txt [hunk 1]
```diff
@@ -2,5 +2,7 @@ This is our test document for exploring merge conflicts.
 
 Please put your name below any existing names ...
 
+Grant Buchanan
+
```
Comment (Bansenauer-Cascadia) [inline sign-in.txt]: This looks good but in the wrong place

**Comment by Teylan** [comment]
I have been eternally shamed

**Comment by Teylan** [comment]
Pick me next :)

**Comment by Teylan** [comment]
:)

**Comment by Bansenauer-Cascadia** [comment]
Hey Grant - I'll take the next person who pulls develop, merges and pushes their branch. Could be you :)

**Comment by Teylan** [comment]
Resolved conflict

File: sign-in.txt [hunk 2]
```diff
@@ -2,7 +2,10 @@ This is our test document for exploring merge conflicts.
 
 Please put your name below any existing names ...
 
+
  - Brian
  - Silvio
  - Stephen
  - Victoria
+ - Grant
```
Comment (Bansenauer-Cascadia) [inline sign-in.txt]: Nicely done

**Pull request merged**